python/cugraph/tests/test_k_truss_subgraph.py

```diff
@@ -47,22 +47,27 @@ def ktruss_ground_truth(graph_file):
     return df
 
 
-def cugraph_k_truss_subgraph(graph_file, k):
+def cugraph_k_truss_subgraph(graph_file, k, directed):
     cu_M = utils.read_csv_file(graph_file)
-    G = cugraph.DiGraph()
+    if directed:
```

Comment:
I was confused about this at first since the doc says `cugraph_k_truss_subgraph` gets a `cuGraph.Graph` and returns a `cuGraph.Graph`. Can we add a comment explaining why the `directed` parameter and `DiGraph` get involved here? I think this would apply to other algos as well.